GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -92,6 +92,7 @@ public abstract class GVFSPlatformConstants
             public static readonly char PathSeparator = Path.DirectorySeparatorChar;
             public abstract string ExecutableExtension { get; }
             public abstract string InstallerExtension { get; }
+            public abstract string EnlistmentDataRoot { get; }
```

Comment:
Naming suggestion:  `EnlistmentDataRoot` is a bit vague.  

Could we stick with a name similar to the old name (e.g. `DotVFSForGitRoot`, `DotVFSForGitName`, or even `DotGVFSRoot`)?  

This would be consistent with the name of the `.git` folder (`GVFSConstants.DotGit.Root`).

Reply:
I think I like `DotVFSForGitRoot` the most, so will use that.

Reply:
Actually, `DotGVFSRoot` is better, to match the `GVFSConstants.DotGVFS` class.